.github/dependabot.yml

```diff
@@ -22,5 +22,3 @@ updates:
       - "gradle"
     commit-message:
       prefix: "[PF-2983]"
-    ignore:
-      - dependency-name: "org.springframework.boot:spring-boot-gradle-plugin"
```

Comment:
This ignore was actually not doing what I hoped, and thus we did update spring-boot… which ended up working.